apps/betterangels/app.config.js

```diff
@@ -129,6 +129,13 @@ export default {
             'Allow $(PRODUCT_NAME) to use your location to log where client interactions take place.',
         },
       ],
+      [
+        'react-native-vision-camera',
+        {
+          cameraPermissionText: '$(PRODUCT_NAME) needs access to your Camera.',
+          enableCodeScanner: true,
```

Comment:
This permission request would get us rejected from the app store if merged.  We need to explain what the permission is for like here: https://github.com/BetterAngelsLA/monorepo/blob/35d7f78ec6ec5ebd9f3f53c5af469ffafbb8033b/apps/betterangels/app.config.js#L122